### PR TITLE
recipe for teneto

### PR DESCRIPTION
**Pull request by mediocretech**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mediocretech** [comment]
@conda-forge/help-python, ready for review!

File: recipes/teneto/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/wiheto/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: b74daefe65c34f200b292eb3cd21d67f357fbd892c7b678c35e772af2b1eb1a2
+
+build:
+  script: python -m pip install . -vv
+  number: 0
+  skip: True  # [win]
```
Comment (ocefpaf) [inline recipes/teneto/meta.yaml:15]: Why not noarch?

Comment (mediocretech) [reply recipes/teneto/meta.yaml:15]: Package is doing some file opening that has a unicode issue

Comment (mediocretech) [reply recipes/teneto/meta.yaml:15]: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1100738&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de

Comment (ocefpaf) [reply recipes/teneto/meta.yaml:15]: That is (usually) fine. The final package may still be functional on Windows. If not, the alternative is to make a OS-noarch. See https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies

I believe this one is OK with the first option.

Comment (mediocretech) [reply recipes/teneto/meta.yaml]: Works for me!  It's noarch again

File: recipes/teneto/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wiheto/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: b74daefe65c34f200b292eb3cd21d67f357fbd892c7b678c35e772af2b1eb1a2
+
+build:
+  noarch: python
+  script: python -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/teneto/meta.yaml:14]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
```

**Pull request merged**